Analyzer/interface/TupleMaker.h

```diff
@@ -743,7 +743,15 @@ void TupleMaker::initializeTuple(Tuple *&tuple,
   
   tuple->Calibration_GiTemplate =  dir.make<TH3F>("Calibration_GiTemplate", ";Module geometry;Path lenght (cm?);Path normalised charge (ke)", 15, 1.0, 16.0, 42, 0.2, 1.6, 500, 0.0, 5000.0);
   tuple->Calibration_GiTemplate_noL1 = dir.make<TH3F>("Calibration_GiTemplate_noL1", ";Module geometry;Path lenght (cm?);Path normalised charge (ke/mm?)", 15, 1.0, 16.0, 42, 0.2, 1.6, 500, 0.0, 5000.0);
-
+  
+  //is it good for me to pass puTreatment_ as new argument in the functions ?
+
+  tuple->Calibration_GiTemplate_PU_1 =  dir.make<TH3F>("Calibration_GiTemplate_PU_1", ";Module geometry;Path lenght (cm?);Path normalised charge (ke)", 15, 1.0, 16.0, 42, 0.2, 1.6, 500, 0.0, 5000.0);
```

Comment:
```suggestion
  tuple->Calibration_GiTemplate_PU_1 =  dir.make<TH3F>("Calibration_GiTemplate_PU_1", ";Module geometry;Path lenght (cm);Path normalised charge (ke/mm)", 15, 1.0, 16.0, 42, 0.2, 1.6, 500, 0.0, 5000.0);
```
is this right? @DenkMybu or is it (ke/cm)? or the length is in (mm) too?